File: MVCProject/app/src/main/java/com/example/mvc_project/MainActivity.java
package com.example.mvc_project;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.example.mvc_project.allproducts.controller.AllProductsActivity;
import com.example.mvc_project.favproducts.controller.FavProductsActivity;

public class MainActivity extends AppCompatActivity {

    Button btAll ;
    Button btFav ;
    Button btEnd;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btAll = findViewById(R.id.btn_allproudcts);
        btFav = findViewById(R.id.btn_favProduct);
        btEnd = findViewById(R.id.btn_finish);


        btAll.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, AllProductsActivity.class);
                startActivity(intent);
            }
        });

        btFav.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, FavProductsActivity.class);
                startActivity(intent);
            }
        });



        btEnd.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();
            }
        });
    }
}

File: MVVM/app/src/main/java/com/example/mvvm/MainActivity.kt
package com.example.mvvm

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import com.example.mvvm.allProducts.view.AllProductsActivity
import com.example.mvvm.favProducts.view.FavProducts

class MainActivity : AppCompatActivity() {
    lateinit var all :Button
    lateinit var exit:Button
    lateinit var fav:Button
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        all = findViewById(R.id.btn_allproudcts)
        exit = findViewById(R.id.btn_finish)
        fav = findViewById(R.id.btn_favProduct)
        all.setOnClickListener {
            var intent = Intent(this,AllProductsActivity::class.java)
            startActivity(intent)
        }
        exit.setOnClickListener {
            finish()
        }

        fav.setOnClickListener {
            var intent = Intent(this,FavProducts::class.java)
            startActivity(intent)
        }
    }
}